### DISCUSSION TITLE
NOTEBOOk 04.02 - DATA CLEANING

### DISCUSSION THREAD

**Discussion by JGranda11** [rramosp/ai4eng.v1#304] [04 - GESTIÓN DE DATOS]
Saludos, 
Lo que pasa es que tengo problemas con la función para mostrar las graficas, de las variables del Dataset que tienen datos faltantes, 
![image](https://github.com/rramosp/ai4eng.v1/assets/112766584/af262779-0956-4341-85b7-8460513aa960)
Al parecer la función ya no esta en uso, y además de eso no alcanza a generar todas las graficas
![image](https://github.com/rramosp/ai4eng.v1/assets/112766584/8c076ff9-c5d3-4171-bd42-848d8942e66c)

**Comment by Felipe-RA** [comment]
Gracias por el heads-up @JGranda11 

Ya estamos trabajando en arreglarlo

**Comment by Felipe-RA** [comment]
Arreglado! @JGranda11 

Lanza nuevamente el código y podrás observar que ya funciona perfecto sin warnings e imprime todo 😎 

Gracias por la paciencia!
